lib/widgets/content.dart

```diff
@@ -406,12 +406,13 @@ class CodeBlock extends StatelessWidget {
 
   final CodeBlockNode node;
 
-  static final _borderColor = const HSLColor.fromAHSL(0.15, 0, 0, 0).toColor();
-
   @override
   Widget build(BuildContext context) {
     return _CodeBlockContainer(
-      borderColor: _borderColor,
+      // (The other caller, MathBlock, passes a nontransparent color.)
+      // TODO(#46) remove comment about MathBlock when TeX is implemented;
+      //   presumably it won't apply anymore.
+      borderColor: Colors.transparent,
```

Comment:
Does this even need a comment? Presumably we'll remove the `borderColor` parameter itself when that happens.